src/component/feature/FeatureToggleList/FeatureToggleListTable.tsx

```diff
@@ -174,6 +190,21 @@ export const FeatureToggleListTable: VFC = () => {
         setStoredParams({ id: sortBy[0].id, desc: sortBy[0].desc || false });
     }, [sortBy, globalFilter, setSearchParams, setStoredParams]);
 
+    const [scrollIndex, setScrollIndex] = useState(0);
+    useEffect(() => {
+        const handleScroll = () => {
+            requestAnimationFrame(() => {
+                const position = window.pageYOffset;
+                setScrollIndex(Math.floor(position / (rowHeight * 5)) * 5);
```

Comment:
This is a bit obfuscated. "Change value every 5 rows scrolled"

Reply:
Could be clearer, yeah.